sdk/src/types/public/anchor-types.ts

```diff
@@ -32,10 +32,13 @@ const IDL = WhirlpoolIDL as Idl;
 export const WHIRLPOOL_CODER = new BorshAccountsCoder(IDL);
 
 /**
- * Size of the Whirlpool account in bytes.
- * @category Solana Accounts
+ * Get the size of an account owned by the Whirlpool program in bytes.
+ * @param accountName Whirlpool account name
+ * @returns Size in bytes of the account
  */
-export const WHIRLPOOL_ACCOUNT_SIZE = WHIRLPOOL_CODER.size(IDL.accounts![4]);
```

Comment:
Since it is an externally disclosed constant, how about deprecating it for the next version ?

```
 * @deprecated Please use {@link getAccountSize} instead.
 * ...
 */
export const WHIRLPOOL_ACCOUNT_SIZE = getAccountSize(AccountName.Whirlpool);
```